.github/workflows/ci.yml

```diff
@@ -117,7 +127,7 @@ jobs:
         # Here support the latest three minor releases of Kubernetes, this can be considered to be roughly
         # the same as the End of Life of the Kubernetes release: https://kubernetes.io/releases/
         # Please remember to update the CI Schedule Workflow when we add a new version.
-        k8s: [ v1.26.0, v1.27.3, v1.28.0 ]
+        k8s: [v1.26.0, v1.27.3, v1.28.0]
```

Comment:
hi, does this line and above `version: '3.x'` line also reported as typo problems by this tool ?

Reply:
No. It's automatically formatted by the extension.

Should I keep it as it was?

Reply:
No, I have checked that your this format is more standardized by refering to [official demo](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)

Besides, thanks very much again for your work~